design1.py

```diff
@@ -0,0 +1,81 @@
+# Time Complexity : O(N/k) k=fixed size of hashmap. N=no. of unique keys stored in hashmap
```

Comment:
The worst case Time Complexity will be O(n^2). If all keys are stored in one single index and remove would cause shifting.